### PR TITLE
Add extension for navigation buttons in left menu

### PR DESCRIPTION
**Pull request by alexabird**

### CODE REVIEW

**Comment by alexabird** [comment]
build now

**Comment by pvargas** [comment]
Should we add an extension point and add these changes downstream instead? 
These changes might not be desired by everyone.

**Comment by alexabird** [comment]
build now

**Review by pvargas — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/app/app.tsx [hunk 1]
```diff
@@ -620,6 +651,8 @@ const SideBar = () => {
           className="h-full w-full"
           wrap="nowrap"
         >
+          {Extensions.includeNavigationButtons && <SideBarNavigationButtons />}
+          <Divider />
```
Comment (andrewkfiedler) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/app/app.tsx]: Should we wrap the divider in this if statement as well?

**Comment by alexabird** [comment]
build now

**Review by andrewkfiedler — APPROVED** [review]

**Pull request merged**